Cargo.toml

```diff
@@ -7,3 +7,7 @@ members = [
         "lalrpop-util",
         "lalrpop",
 ]
+
+[workspace.package]
+edition = "2021"
+rust-version = "stable"
```

Comment:
Note that this just a "virtual" declaration of the fields, and they have to be inherited by each package of the workspace: <https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table>

Reply:
Thank you! I also found `stable` is not a valid value.